From 90455984f0fcb1a3a4ea3c3510e9d35f89fb2178 Mon Sep 17 00:00:00 2001
From: Oliver Smith <ollieparanoid@postmarketos.org>
Date: Tue, 27 Jun 2023 09:20:53 +0200
Subject: [PATCH] Remove test_process_value_error

Remove the test, as it is failing in Alpine and has been removed
upstream too (in master, not yet in latest release):
https://github.com/netromdk/vermin/commit/bda5ae6c770d24da666ee10e46a71bfa4f8cf623

Create a separate patch as the upstream patch doesn't apply cleanly on
the release tag.
---
 tests/general.py | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/tests/general.py b/tests/general.py
index 65ea020..f7857c3 100644
--- a/tests/general.py
+++ b/tests/general.py
@@ -811,18 +811,6 @@ test.py:6:9:2.7:3.2:'argparse' module
     self.assertEmpty(proc_res.text)
     self.assertEmpty(proc_res.bps)
 
-  # This test is ignored on Windows for now because it does not yield the value error that other
-  # platforms do.
-  @VerminTest.skipPlatform("win32")
-  def test_process_value_error(self):
-    # (Py3) ValueError: source code string cannot contain null bytes
-    # (Py2) TypeError: compile() expected string without null bytes
-    fp = ScopedTemporaryFile()
-    fp.write(b'\0')
-    fp.close()
-    proc_res = process_individual((fp.path(), self.config))
-    self.assertEqual(proc_res, None)
-
   def test_process_invalid_versions(self):
     with ScopedTemporaryFile() as fp:
       fp.write(b"""long(42)
-- 
2.41.0

